api/configs/middleware/cache/redis_config.py

```diff
@@ -68,3 +68,18 @@ class RedisConfig(BaseSettings):
         description="Socket timeout in seconds for Redis Sentinel connections",
         default=0.1,
     )
+
+    REDIS_USE_CLUSTERS: Optional[bool] = Field(
```

Comment:
```suggestion
    REDIS_USE_CLUSTERS: bool = Field(
```
`Optional` is not required as default value is always provided.